ElectronClient/InteropServiceHelper.js

```diff
@@ -107,9 +108,14 @@ class InteropServiceHelper {
 		return this.exportNoteTo_('printer', noteId, options);
 	}
 
-	static async defaultFilename(noteId, fileExtension) {
-		if (!noteId) return '';
-		const note = await Note.load(noteId);
+	static async defaultFilename(noteIds, fileExtension) {
+		if (!noteIds) {
+			const date = time.formatMsToLocal(new Date().getTime(), time.dateFormat());
+			return friendlySafeFilename(`${date} - ${_('Joplin')}.${fileExtension}`);
```

Comment:
I think just `${date}.${fileExtension}` would be enough as the "Joplin" part is not relevant once the file has been exported (it's just a Markdown or PDF file like any other).

Reply:
Yes good point, I've updated it now.